--- tgif-QPL-4.2/Imakefile~	2009-10-07 16:02:29.000000000 +0200
+++ tgif-QPL-4.2/Imakefile	2009-10-07 16:02:19.000000000 +0200
@@ -344,7 +344,14 @@
  * NormalRelocatableTarget(frontend11,$(FRONTENDOBJS))
  */
 
+ICONSDIR =icons/
+
 MakeDirectories(install,$(TGIFDIR))
+MakeDirectories(install,$(TGIFDIR)/hicolor/22x22)
+MakeDirectories(install,$(TGIFDIR)/hicolor/32x32)
+MakeDirectories(install,$(TGIFDIR)/hicolor/48x48)
+MakeDirectories(install,$(TGIFDIR)/hicolor/64x64)
+MakeDirectories(install,$(TGIFDIR)/hicolor/128x128)
 InstallNonExec(tgif.Xdefaults,$(TGIFDIR))
 InstallNonExec(tgificon.eps,$(TGIFDIR))
 InstallNonExec(tgificon.obj,$(TGIFDIR))
@@ -379,7 +386,6 @@
 
 SPICEDIR =spice/
 XBMDIR =xbm/
-ICONSDIR =icons/
 RMCASTDIR_QPL =rmcast-qpl
 RMCASTDIR_GPL =rmcast-gpl
 
