--- tgif-QPL-4.2.1/Imakefile~	2009-10-08 17:27:51.000000000 +0000
+++ tgif-QPL-4.2.1/Imakefile	2009-10-14 15:27:45.000000000 +0000
@@ -347,6 +347,11 @@
 ICONSDIR =icons/
 
 MakeDirectories(install,$(TGIFDIR))
+MakeDirectories(install,$(TGIFDIR)/hicolor/22x22)
+MakeDirectories(install,$(TGIFDIR)/hicolor/32x32)
+MakeDirectories(install,$(TGIFDIR)/hicolor/48x48)
+MakeDirectories(install,$(TGIFDIR)/hicolor/64x64)
+MakeDirectories(install,$(TGIFDIR)/hicolor/128x128)
 InstallNonExec(tgif.Xdefaults,$(TGIFDIR))
 InstallNonExec(tgificon.eps,$(TGIFDIR))
 InstallNonExec(tgificon.obj,$(TGIFDIR))
